modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -626,20 +638,130 @@
+                    .to receive(:get_by_id_and_icn).and_return(nil)
+
+                  get claim_by_id_path, headers: auth_header
+
+                  json_response = JSON.parse(response.body)
+                  expect(response.status).to eq(200)
+                  expect(json_response).to be_an_instance_of(Hash)
+                  expect(json_response['claimType']).to eq('value from BGS')
+                  expect(json_response['status']).to eq('PENDING')
+                end
+              end
+            end
+          end
+        end
+      end
+
+      describe "handling the 'supporting_documents'" do
+        context 'it has documents' do
+          let(:bgs_claim) do
+            {
+              benefit_claim_details_dto: {
+                benefit_claim_id: '111111111',
+                claim_status_type: 'value from BGS',
+                bnft_claim_lc_status: [{
+                  phas_chngd_dt: DateTime.now,
+                  phase_type: 'Pending'
+                }, {
+                  phas_chngd_dt: DateTime.now - 1.day,
+                  phase_type: 'Initial Review'
+                }]
+              }
+            }
+          end
+
+          it "returns a claim with 'supporting_documents'" do
+            with_okta_user(scopes) do |auth_header|
+              VCR.use_cassette('bgs/tracked_items/find_tracked_items') do
+                VCR.use_cassette('evss/documents/get_claim_documents') do
+                  expect_any_instance_of(BGS::EbenefitsBenefitClaimsStatus)
+                    .to receive(:find_benefit_claim_details_by_benefit_claim_id).and_return(bgs_claim)
+                  expect(ClaimsApi::AutoEstablishedClaim)
+                    .to receive(:get_by_id_and_icn).and_return(nil)
+
+                  get claim_by_id_path, headers: auth_header
+
+                  json_response = JSON.parse(response.body)
+                  first_doc_id = json_response.dig('supportingDocuments', 0, 'documentId')
+                  expect(response.status).to eq(200)
+                  expect(json_response).to be_an_instance_of(Hash)
+                  expect(json_response['claimType']).to eq('value from BGS')
+                  expect(first_doc_id).to eq('{54EF0C16-A9E7-4C3F-B876-B2C7BEC1F834}')
+                end
+              end
+            end
+          end
+        end
+
+        context 'it has no documents' do
+          let(:bgs_claim) do
+            {
+              benefit_claim_details_dto: {
+                benefit_claim_id: '222222222',
+                claim_status_type: 'value from BGS',
+                bnft_claim_lc_status: [{
+                  phas_chngd_dt: DateTime.now,
+                  phase_type: 'Pending'
+                }, {
+                  phas_chngd_dt: DateTime.now - 1.day,
+                  phase_type: 'Initial Review'
+                }]
+              }
+            }
+          end
+
+          it "returns a claim with 'supporting_documents'" do
```

Comment:
*nitpick
If the spec name were `it "returns a claim with 'suporting_documents' as an empty array'" do`, that would be more descriptive and easier to reason in the future.

Not a big deal though.  I'll let you decide if you wanna change it.